diff --git a/dist/index.mjs b/dist/index.mjs
index 67e1902b1a15b8d4a23cb1a7504300b4c57ac6c0..bc23369761d5f43b2a5f2bff885f2e7d757fdded 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -29,7 +29,7 @@ function codspeedPlugin() {
           pool: "forks",
           poolOptions: {
             forks: {
-              execArgv: getV8Flags()
+              execArgv: [...getV8Flags(), '--enable-source-maps']
             }
           },
           runner: getCodSpeedFileFromName("runner"),